junit5-jupiter-starter-ant/build.sh

```diff
@@ -15,9 +16,16 @@ curl --remote-name "https://archive.apache.org/dist/ant/binaries/${ant_archive}"
 tar --extract -z --exclude "${ant_folder}/manual" --file "${ant_archive}"
 
 #
-# Load and store junit-platform-console-standalone jar into ${ANT_HOME}/lib.
+# Load and store necessary JUnit5 jar into ${ANT_HOME}/lib.
 #
-(cd "${ant_folder}/lib" && curl --remote-name  "http://central.maven.org/maven2/org/junit/platform/junit-platform-console-standalone/${junit_platform_version}/junit-platform-console-standalone-${junit_platform_version}.jar")
+(cd "${ant_folder}/lib" && curl --remote-name "http://central.maven.org/maven2/org/junit/platform/junit-platform-commons/${junit_platform_version}/junit-platform-commons-${junit_platform_version}.jar")
+(cd "${ant_folder}/lib" && curl --remote-name "http://central.maven.org/maven2/org/junit/platform/junit-platform-engine/${junit_platform_version}/junit-platform-engine-${junit_platform_version}.jar")
+(cd "${ant_folder}/lib" && curl --remote-name "http://central.maven.org/maven2/org/junit/platform/junit-platform-launcher/${junit_platform_version}/junit-platform-launcher-${junit_platform_version}.jar")
+(cd "${ant_folder}/lib" && curl --remote-name "http://central.maven.org/maven2/org/junit/jupiter/junit-jupiter-api/${junit_jupiter_version}/junit-jupiter-api-${junit_jupiter_version}.jar")
+(cd "${ant_folder}/lib" && curl --remote-name "http://central.maven.org/maven2/org/junit/jupiter/junit-jupiter-engine/${junit_jupiter_version}/junit-jupiter-engine-${junit_jupiter_version}.jar")
+(cd "${ant_folder}/lib" && curl --remote-name "http://central.maven.org/maven2/org/junit/jupiter/junit-jupiter-params/${junit_jupiter_version}/junit-jupiter-params-${junit_jupiter_version}.jar")
+(cd "${ant_folder}/lib" && curl --remote-name "http://central.maven.org/maven2/org/opentest4j/opentest4j/1.2.0/opentest4j-1.2.0.jar")
+(cd "${ant_folder}/lib" && curl --remote-name "http://central.maven.org/maven2/org/apiguardian/apiguardian-api/1.1.0/apiguardian-api-1.1.0.jar")
```

Comment:
Wouldn't it be easier to use [Apache Ivy](https://ant.apache.org/ivy/) to retrieve the necessary dependencies instead of handling that manually with `curl`?

Reply:
How about supporting Ivy when supporting [Ant-wrapper](https://github.com/rimerosolutions/ant-wrapper#using-apache-ivy-recommended) at #66 ?

Reply:
Why not keeping `junit-platform-console-standalone` JAR here? Does it no longer work with the `<junitlauncher>` task?

Reply:
> How about supporting Ivy when supporting [Ant-wrapper](https://github.com/rimerosolutions/ant-wrapper#using-apache-ivy-recommended) at #66 ?

Does the Ant-wrapper make it somehow easier to use Ivy?

Reply:
> Why not keeping `junit-platform-console-standalone` JAR here? Does it no longer work with the `<junitlauncher>` task?

Ant allows testing to be launched, so junit-platform-console-standalone is not required.
https://ant.apache.org/manual/Tasks/junitlauncher.html

Reply:
> > How about supporting Ivy when supporting [Ant-wrapper](https://github.com/rimerosolutions/ant-wrapper#using-apache-ivy-recommended) at #66 ?
> 
> Does the Ant-wrapper make it somehow easier to use Ivy?

I don't know much the Ant-wrapper. Is it better to use ivy directly?

Reply:
In my opinion, it's always better to use Ant _with_ Ivy, since Ivy provides dependency management similar to that provided by Maven or Gradle.

From the documentation for the Ant-wrapper, it appears that that's easier to set up if you're already using Ivy.

So maybe we should actually take the following steps.

1. Use Ivy in the Ant examples.
2. Use the Ant-wrapper in the Ant examples.
3. Use the `junit-launcher` task in the Ant examples.

Thoughts?

Reply:
I see, sounds good! but what does mean No.3? I already use this one https://github.com/junit-team/junit5-samples/pull/114/commits/fb685f486d7d021513508b07907c6491c3196d49#diff-72ebd09ba035959b381f450ac03a68efR37

Reply:
I meant that we should _ideally_ follow those steps.

But... if we mix up the order a bit and still arrive at the same outcome, that works, too. 😉